classes/Attribute.php

```diff
@@ -213,29 +255,32 @@ public static function checkAttributeQty($id_product_attribute, $qty, Shop $shop
      * Update array with veritable quantity
      *
      * @deprecated since 1.5.0
-     * @param array &$arr
-     * @return bool
+     *
+     * @param array $arr Array with Product ID and quantity
+     *
+     * @return bool Whether the quantity of the product in the given array could be updated successfully
      */
     public static function updateQtyProduct(&$arr)
```

Comment:
I think that this method can by completely removed as it is deprecated for a while